dref/models.py

```diff
@@ -78,20 +78,19 @@ class IdentifiedNeed(models.Model):
     class Title(models.TextChoices):
         SHELTER_HOUSING_AND_SETTLEMENTS = "shelter_housing_and_settlements", _("Shelter Housing And Settlements")
         LIVELIHOODS_AND_BASIC_NEEDS = "livelihoods_and_basic_needs", _("Livelihoods And Basic Needs")
+        MULTI_PURPOSE_CASH_GRANTS = "multi_purpose_cash_grants", _("Multi purpose cash grants")
         HEALTH = "health", _("Health")
         WATER_SANITATION_AND_HYGIENE = "water_sanitation_and_hygiene", _("Water, Sanitation And Hygiene")
         PROTECTION_GENDER_AND_INCLUSION = "protection_gender_and_inclusion", _("Protection, Gender And Inclusion")
         EDUCATION = "education", _("Education")
-        MIGRATION = "migration", _("Migration")
-        MULTI_PURPOSE_CASH_GRANTS = "multi_purpose_cash_grants", _("Multi purpose cash grants")
+        MIGRATION_AND_DISPLACEMENT = "migration_and_displacement", _("Migration And Displacement")
         RISK_REDUCTION_CLIMATE_ADAPTATION_AND_RECOVERY = "risk_reduction_climate_adaptation_and_recovery", _(
             "Risk Reduction, Climate Adaptation And Recovery"
         )
-        COMMUNITY_ENGAGEMENT_AND_ACCOUNTABILITY = "community_engagement_and _accountability", _(
+        COMMUNITY_ENGAGEMENT_AND_ACCOUNTABILITY = "community_engagement_and_accountability", _(
             "Community Engagement And Accountability"
         )
-        ENVIRONMENT_SUSTAINABILITY = "environment_sustainability ", _("Environment Sustainability")
-        SHELTER_CLUSTER_COORDINATION = "shelter_cluster_coordination", _("Shelter Cluster Coordination")
```

Comment:
We will need to migrate data as well.

Reply:
@frozenhelium @udaynwa 
Can you confirm the behaviour here?

Currently, this enums are used by title which are non-nullable field. We have 3 options.

- Ignore... Will raise error on runtime
- Migrate to another value
- Delete IdentifiedNeed objects related to this value

Let's figure out the best case.

Reply:
Let's do the option (c) "Delete `IdentifiedNeed` objects related to that value"

Reply:
Removed `IdentifiedNeed`  related objects.